INN/umbrella-sfpublicpress
# Styling for homepage audio top section widgets and podcasts widget

## Changes

This pull request makes the following changes:

- Adds styles for homepage text widgets in the `homepage bottom` widget area to match the designs for the homepage audio top section
- Adds a podcast widget and relevant styling

Desktop:
![screencapture-sfpp-test-2020-01-14-10_23_30](https://user-images.githubusercontent.com/18353636/72356867-fe34a700-36b7-11ea-8276-d3d232b9051e.png)

Mobile:
![screencapture-sfpp-test-2020-01-14-10_23_38](https://user-images.githubusercontent.com/18353636/72356894-0f7db380-36b8-11ea-8dac-9187f68f2afb.png)

## Why



For #22 and #23 

## Testing/Questions

Features that this PR affects:

- Homepage audio top section
- Podcasts widget
- `widget-content.php` partial


Questions that need to be answered before merging:

- [ ] The image looks pretty bad on mobile. We should probably find a better quality one to replace it with.

Steps to test this PR:

1. Add a text widget to the homepage bottom widget area. Add `Audio` as the title and give it a title link and add this as the text:
```
<h3 class="orange">Our Latest "Civic" Podcast Episode</h3>

Embed goes here
&nbsp;
<hr/>
```
2. Add another text widget to the homepage bottom widget area with no title. Add this text:
```
<h3 class="orange">Our Low-Power Community Radio Station</h3>
<img class="alignleft wp-image-110 size-thumbnail" src="http://sfpp.test/wp-content/uploads/2020/01/Image-133@2x-140x140.png" alt="" width="140" height="140" />KSFP, our low-power community radio station, broadcasts on 102.5 FM from San Francisco’s Sutro Tower 12 hours a day, featuring Public Press reporting and public radio programs.

<a class="view-more-link" href="#">Learn more about KSFP</a>

&nbsp;
&nbsp;
<hr/>
```
Here's the image to use:
![Image-133@2x](https://user-images.githubusercontent.com/18353636/72292677-ddb90e00-361f-11ea-8b53-7f5004f67758.png)
3. Add the new `San Francisco Public Press Podcasts` widget to the homepage bottom widget area with no title, 3 posts, no excerpt, no duplicates, and `More Podcasts` more link
4. View the frontend and verify all of it looks ok on desktop and mobile
